core/payment/examples/README.md

```diff
@@ -10,6 +10,8 @@ cargo run --example payment_api
 ```
 To use GNT instead of dummy driver us `cargo run --example payment_api -- --driver=gnt` instead.
```

Comment:
Could you update this line as well? It should be like this:
ERC-20 driver on rinkeby: `cargo run --example payment_api -- --driver=erc20 --platform=erc20-rinkeby-tglm`
ERC-20 driver on mainnet: `cargo run --example payment_api -- --driver=erc20 --network=mainnet --platform=erc20-mainnet-glm`
ZkSync driver on rinkeby: `cargo run --example payment_api -- --driver=zksync --platform=zksync-rinkeby-tglm`
ZkSync driver on mainnet: `cargo run --example payment_api -- --driver=zksync --network=mainnet --platform=zksync-mainnet-glm`

The `--platform` should be passed to `invoice_flow`/`debit_not_flow` as well.